src/ui/canvas/gdi/GdiPlusBitmap.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright_License {
+
+  XCSoar Glide Computer - http://www.xcsoar.org/
+  Copyright (C) 2000-2021 The XCSoar Project
+  A detailed list of copyright holders can be found in the file "AUTHORS".
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+}
+*/
+
+#include "GdiPlusBitmap.hpp"
+
+#if defined(_MSC_VER)
+# include <algorithm>
+# define min std::min
+# define max std::max
+#endif  // _MSC_VER
+
+#include <unknwn.h>
+#include <gdiplus.h>
+
+using namespace Gdiplus;
+
+// can load: BMP, GIF, JPEG, PNG, TIFF, Exif, WMF, and EMF
+HBITMAP GdiLoadImage(const TCHAR* filename)
+{
+  HBITMAP result =nullptr;
+  Bitmap bitmap(filename, false);
+  bitmap.GetHBITMAP(0, &result);
```

Comment:
Does this even compile?
https://docs.microsoft.com/en-us/windows/win32/api/gdiplusheaders/nf-gdiplusheaders-bitmap-gethbitmap
> Status GetHBITMAP(
>  const Color & colorBackground,
>  HBITMAP       *hbmReturn
>);

How can you pass `0` for the `const Color&`?